CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - [#1467](https://github.com/openshift/cluster-monitoring-operator/pull/1467) Add bodysize limit for metric scraping
 - [#1661](https://github.com/openshift/cluster-monitoring-operator/pull/1661) Support deployment of dedicated Alertmanager for user-defined alerts.
 - [#1682](https://github.com/openshift/cluster-monitoring-operator/pull/1682) Support AlertmanagerConfig v1beta1.
+- [#1639](https://github.com/openshift/cluster-monitoring-operator/pull/1639) Increase "for" duration to 15min in KubeAggregatedAPIDown alert.
```

Comment:
Let remove the changelog entry, we usually track feature addition here. The bug fix will be added to the release notes by the doc team.